src/status_im2/contexts/onboarding/create_password/view.cljs

```diff
@@ -214,13 +212,13 @@
        :accessible false}
       [rn/view {:style style/flex-fill}
        [rn/keyboard-avoiding-view {:style style/flex-fill}
-        [navigation-bar/navigation-bar
-         {:stack-id              :new-to-status
-          :top                   top
-          :right-section-buttons [{:type            :grey
-                                   :icon-background :blur
-                                   :icon            :i/info
-                                   :on-press        on-press-info}]}]
+        [quo/page-nav
+         {:margin-top top
```

Comment:
is `margin-top` actually passed here? Should it be passed in a container-style prop for consistency with other components? 🤔